pyxrf/view/lineplot.enaml

```diff
@@ -108,6 +129,9 @@ enamldef PlotMain(DockItem):
                                             param_model.auto_fit_all)
                     except (AttributeError, ValueError):
                         pass
+                    if plot_model.plot_exp_opt:
+                        plot_model.plot_exp_opt = False
+                        plot_model.plot_exp_opt = True
```

Comment:
Why is it needed to be assigned twice? Some kind of reset?

Reply:
Yes, setting it twice makes the plot to redraw no matter what the current state of '`plot_exp_opt` is. The value must change in order for the observer function to be called.